libraries/AP_ICEngine/AP_ICEngine.h

```diff
@@ -122,12 +122,17 @@ class AP_ICEngine {
 
     enum class Options : uint16_t {
         DISABLE_IGNITION_RC_FAILSAFE=(1U<<0),
+        REDLINE_WARNING = (1U << 0),
```

Comment:
```suggestion
        REDLINE_WARNING = (1U << 1),
```

Plus, this should be DISABLE_REDLINE_WARNING

Reply:
Fixed. I had it as an enable before I remembered that option parameters are usually disables rather than enables.